src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -808,24 +929,26 @@ export async function getAccountLiquidity(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: GetAccountLiquidityRequest
-): Promise<string> {
+): Promise<GetAccountLiquidityResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const liquidity = await panopticish.getAccountLiquidity(
+  const result = await panopticish.getAccountLiquidity(
     wallet,
     req.univ3pool,
     req.owner,
     req.tokenType,
     req.tickLower,
     req.tickUpper
   );
-  return liquidity;
+  return {
+    accountLiquidities: result.accountLiquidities
+  };
 }
 
 export async function getAccountPremium(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: GetAccountPremiumRequest
-): Promise<string> {
+): Promise<GetAccountPremiumResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const liquidity = await panopticish.getAccountPremium(
```

Comment:
Let's also change rename `liquidity` var to `accountPremiumData`

Reply:
Done: c2f20b75c082581ff451ed5b75b3e804f7fdfc05